src/util/UserInfoHelperFunctions.ts

```diff
@@ -70,3 +70,7 @@ export function canAccessTeam(hacker?: IHacker): boolean {
     status === HackerStatus.HACKER_STATUS_CHECKED_IN
   );
 }
+
+export function canAccessBus(hacker?: IHacker): boolean {
+  return hacker ? Boolean(hacker.needsBus) : false;
```

Comment:
`hacker && hacker.needsBus` for cleaner logic

Reply:
Word, I just got typescript bugs for some reason :( 
![image](https://user-images.githubusercontent.com/17463268/51133789-e3f58e00-1803-11e9-8d48-2283d9c1c623.png)

Reply:
😠 

we can leave it this way then